pkg/mev/pkg.go

```diff
@@ -18,6 +19,21 @@ const (
 	ETHSendBundleMethod         = "eth_sendBundle"
 )
 
+type IBundleSender interface {
+	Send(ctx context.Context, blockNumber uint64, tx ...*types.Transaction) (SendBundleResponse, error)
```

Comment:
Should be `SendBundle`

Reply:
If we combine both the interface name and the function, name we have something like `BundleSender.Send`, I think this is enough information. `BundleSender.SendBundle` has duplication in its name